pkg/cli/login/loginoptions.go

```diff
@@ -266,6 +267,26 @@ func (o *LoginOptions) gatherProjectInfo() error {
 		return err
 	}
 
+	canRequest, err := loginutil.CanRequestProjects(o.Config, o.DefaultNamespace)
+	if err != nil {
+		return err
+	}
+	if !canRequest {
+		msg := ""
+		res, err := projectClient.RESTClient().Get().Resource("projectrequests").DoRaw(context.TODO())
+		if err != nil {
+			status := metav1.Status{}
+			err := json.Unmarshal(res, &status)
+			if err != nil {
+				return err
+			}
+			if len(status.Details.Causes) != 0 && status.Details.Causes[0].Message != "" {
+				msg = status.Details.Causes[0].Message
+			}
```

Comment:
This is where the `project.config.openshift.io/cluster Spec.ProjectRequestMessage` is stored.  If it exists, return it to the NoProjectsExistMessage, if not, 
the default `You don't have any projects. Contact your system administrator to request a project.` will display